src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.directory.server;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.gridsuite.directory.server.dto.AccessRightsAttributes;
+import org.gridsuite.directory.server.dto.DirectoryAttributes;
+import org.gridsuite.directory.server.dto.ElementAttributes;
+import org.gridsuite.directory.server.repository.DirectoryElementEntity;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Optional;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + DirectoryApi.API_VERSION)
+@Api(tags = "directory-server")
+public class DirectoryController {
+
+    private final DirectoryService service;
+
+    public DirectoryController(DirectoryService service) {
+        this.service = service;
+    }
+
+    @PostMapping(value = "/directories/create")
```

Comment:
It does not look like a REST API, this is more a RPC API. For a clean REST API, we need to think about resources and operations on resources.
Why not just a POST on /directories/. POST already means create.

Reply:
I agree. Corrected.